common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -5921,6 +5963,9 @@ static jobjectArray get_X509Type_ext_oids(JNIEnv* env, jlong x509Ref, jint criti
         JNI_TRACE("get_X509Type_ext_oids(%p, %d) => x509 == null", x509, critical);
         return nullptr;
     }
+
+    // TODO(davidben): Remove this check. It should be impossible and will no longer work when
+    // the structs are opaque.
```

Comment:
I'll upload a PR to get rid of this shortly afterwards, but I figure it ought to be a separate PR.

Reply:
👍